Arrowgene.Ddon.Server/Network/PingRequestPacketHandler.cs

```diff
@@ -0,0 +1,50 @@
+using System;
+using System.Timers;
+using Arrowgene.Ddon.Shared.Entity;
+
+namespace Arrowgene.Ddon.Server.Network
+{
+    public abstract class PingRequestPacketHandler<TClient, TReqStruct, TResStruct> : RequestPacketHandler<TClient, TReqStruct, TResStruct>
+        where TClient : Client
+        where TReqStruct : class, IPacketStructure, new()
+        where TResStruct : ServerResponse, new()
+    {
+
+        private static readonly double TIMER_INTERVAL_MS = 60000; // 1 minute
+
+        private readonly Timer TimeoutCheckTimer;
+
+        protected PingRequestPacketHandler(DdonServer<TClient> server) : base(server)
```

Comment:
Isn't this logic only useful in the case of the conection ping handler?
The client/login ping will only be sent once after successful login.
Does this not mean that we are unnecessarily running two timers checking for the same thing for all clients?

Reply:
One for the login server and another for the game server, which each has different clients connected to them. Although yes, we could simplify this into one timer that checks both servers